server/express-router.md

```diff
@@ -0,0 +1,96 @@
+Previously, we've learnt to separate routes into their own file, by exporting them as functions and requiring them within our `server.js` file:
+
+```js
+/*
+ * server.js
+ */
+
+var express = require('express')
+var routes = require('./routes')
+
+var PORT = 3000
+var app = express()
+
+app.get('/', routes.index)
+
+app.listen(PORT, function () {
+  console.log('The server is listening on port', PORT)
+})
+
+/*
+ * routes.js
+ */
+
+module.exports = {
+  index: index
+}
+
+function index (req, res) {
+  res.send('<h1>Hello world</h1>')
+}
+```
+
+However, Express has built in functionality to take care of this also:
+
+### Express Router
+
+We can use `express.Router` to create modular route handlers. The following example creates a router as a module, defines some routes, and mounts the router module on a path in the main app (`server.js`).
+
+Let's say we wanted to create a `/birds` route where we can:
+- view a list of birds via get `/birds`
+- view a single bird by id via get `/birds/:id`
+- add a new bird via post `/birds`
+
+Our code may look like this:
+
+```js
+/*
+ * server.js
+ */
+
+var express = require('express')
+var birds = require('./birds')
+
+var PORT = 3000
+var app = express()
+
+app.use('/birds', birds)
+
+app.listen(PORT, function () {
+  console.log('The server is listening on port', PORT)
+})
+
+/*
+ * birds.js
+ */
+
+var express = require('express')
+var router = express.Router()
+
+router.get('/', function (req, res) {
+  // ...
+})
+
+router.get('/:id', function (req, res) {
+  // ...
+})
+
+router.post('/:id', function (req, res) {
```

Comment:
Because this is a POST (create), we wouldn't have an `:id` yet. I think the route should be `'/'`.